frontend/www/js/omegaup/problem/edit.ts

```diff
@@ -111,18 +113,33 @@ OmegaUp.on('ready', () => {
               name: tagname,
               public: isPublic,
             })
-              .then(response => {
+              .then(() => {
                 ui.success(T.tagAdded);
+                if (isPublic) {
+                  this.selectedPublicTags?.push(tagname);
+                } else {
+                  this.selectedPrivateTags?.push(tagname);
```

Comment:
nit:

```suggestion
                  this.selectedPublicTags.push(tagname);
                } else {
                  this.selectedPrivateTags.push(tagname);
```

como estas listas ya no pueden ser `undefined` (gracias!), se puede eliminar el operador de optional chaining.